pysat/tests/test_sw.py

```diff
@@ -159,10 +166,20 @@ def test_combine_kp_none(self):
     def test_combine_kp_one(self):
         """ Test combine_kp failure when only one instrument is provided"""
 
-        combo_in = {"standard_inst": self.testInst}
+        # Load a test instrument
+        testInst = pysat.Instrument()
+        testInst.data = pds.DataFrame({'Kp': np.arange(0, 4, 1.0/3.0)},
+                                      index=[pysat.datetime(2009, 1, 1)
+                                             + pds.DateOffset(hours=3*i)
+                                             for i in range(12)])
+        testInst.meta = pysat.Meta()
+        testInst.meta.__setitem__('Kp', {testInst.meta.fill_label:
```

Comment:
this should be 
testInst.meta['Kp'] = {testInst.meta.fill_label: np.nan}

Reply:
fixed